### ISSUE TITLE
Create HTML header area

### ISSUE THREAD

**Issue by reginacas** [reginacas/sense-ui#12]
## Description
Currently, the html does not have a <header> element and the h1 and intro description are inside <main>. By changing it into one header element, we improve the semantic structure of the page

## Origin:
Suggested by @paulGeoghegan :

> The navbar and that generall top area should probably be in a element.
